### PR TITLE
Changed the macOS build instructions

### PR DESCRIPTION
**Pull request by JustusAdam**
I tried the build instructions to get IHaskell working.

I documented the results in the changes I made to the build instructions.

I did this install on an almost fresh machine, so it _should_ be complete.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,22 +43,23 @@ ihaskell install --stack
 
 ## Mac
 
-These haven't been tested and there may be some missing required packages. But
-they will be soon.
+You need to have [Homebrew](https://brew.sh) installed. 
+If you do not have it yet run `/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"` in your terminal.
+
+You also need the xcode command line tools.
```
Comment (vaibhavsagar) [inline README.md]: Can you capitalise the X in 'xcode'?

File: README.md [hunk 2]
```diff
@@ -43,22 +43,23 @@ ihaskell install --stack
...
-curl -sSL https://get.haskellstack.org/ | sh
+brew install python3 zeromq libmagic cairo pkg-config haskell-stack pango
 git clone https://github.com/gibiansky/IHaskell
 cd IHaskell
 stack install gtk2hs-buildtools
 stack install --fast
 ihaskell install --stack
 ```
 
+Tested on macOS Sierra (10.12.6)
```
Comment (vaibhavsagar) [inline README.md]: Can you italicise this line?

**Comment by vaibhavsagar** [comment]
Thanks so much! I've left some minor comments and am happy to merge after they are addressed.

**Comment by JustusAdam** [comment]
Happy to make those changes, here you go.

BTW I am really happy that this project exists. 
Notebooks are great and I'm so glad I can now use my favourite language in them.

**Comment by vaibhavsagar** [comment]
Thanks so much!

**Pull request merged**